spec/services/rapid_ready_for_decision/fast_track_pdf_upload_manager_spec.rb

```diff
@@ -4,27 +4,7 @@
 require 'rapid_ready_for_decision/disability_compensation_job'
 
 RSpec.describe RapidReadyForDecision::FastTrackPdfUploadManager do
-  let(:user) { create(:disabilities_compensation_user) }
-  let(:auth_headers) do
-    EVSS::DisabilityCompensationAuthHeaders.new(user).add_headers(EVSS::AuthHeaders.new(user).to_h)
-  end
-  let(:saved_claim) { FactoryBot.create(:va526ez) }
-  let(:original_form_json) do
-    File.read('spec/support/disability_compensation_form/submissions/with_uploads.json')
-  end
-
-  let(:original_form_json_uploads) do
-    JSON.parse(original_form_json)['form526_uploads']
-  end
-
-  let(:form526_submission) do
-    Form526Submission.create(
-      user_uuid: user.uuid,
-      saved_claim_id: saved_claim.id,
-      auth_headers_json: auth_headers.to_json,
-      form_json: original_form_json
-    )
-  end
+  let(:form526_submission) { create(:form526_submission, :hypertension_claim_for_increase_with_uploads) }
```

Comment:
Replace all this with a FactoryBot call.

Reply:
🎉